x-pack/plugins/enterprise_search/public/applications/workplace_search/index.tsx

```diff
@@ -6,14 +6,8 @@
 
 import React, { useContext } from 'react';
 import { Route, Redirect, Switch } from 'react-router-dom';
-import { Provider } from 'react-redux';
-import { Store } from 'redux';
-import { getContext, resetContext } from 'kea';
-
-resetContext({ createStore: true });
-
-const store = getContext().store as Store;
```

Comment:
Another heads up to WS that this was moved to the top-level `public/applications/index.tsx`